pkg/util/constant.go

```diff
@@ -57,9 +58,10 @@ const (
 	NonDBID             = int64(0)
 	InvalidDBID         = int64(-1)
 
-	PrivilegeWord      = "Privilege"
-	PrivilegeGroupWord = "PrivilegeGroup"
-	AnyWord            = "*"
+	PrivilegeWord            = "Privilege"
+	PrivilegeGroupWord       = "PrivilegeGroup"
+	CustomPrivilegeGroupWord = "CustomPrivilegeGroup"
```

Comment:
recommand to reuse the prefix `PrivilegeGroup`, then you can reuse `PrivilegeNameForAPI` and `PrivilegeNameForMetastore` to do transform between privilege name and action name